devel/fetch_metrics/fetch_metrics.py

```diff
@@ -3,16 +3,16 @@
 import argparse
 import json
 import os
-from spikeextractors import RecordingExtractor, SortingExtractor
-from spikecomparison import GroundTruthComparison
-import spiketoolkit as st
 import time
-from typing import Any, Dict, List, TypedDict, Union
-
+from typing import Any, Dict, List, TypedDict, Union, cast
+import spikeextractors as se
 import hither2 as hi
 import kachery_p2p as kp
 import labbox_ephys as le
 
+thisdir = os.path.dirname(os.path.realpath(__file__))
+spiketoolkit_image = hi.DockerImageFromScript(name='magland/spiketoolkit', dockerfile=f'{thisdir}/docker/Dockerfile')
```

Comment:
Here's where we specify the docker/singularity to use. See the hi.Config line below.